system-container/run-system-container.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 
+export ANSIBLE_LOG_PATH=/var/log/ansible.log
```

Comment:
This commit/single line change could/should be squashed into the first commit of the PR: "installer: support running as a system container", the one that introduces this file.